share/eds/testing.go

```diff
@@ -195,9 +199,12 @@ func testSample(
 	t *testing.T,
 	acc Accessor,
 	roots *share.AxisRoots,
-	rowIdx, colIdx int,
+	idx shwap.SampleIndex,
 ) {
-	shr, err := acc.Sample(ctx, rowIdx, colIdx)
+	shr, err := acc.Sample(ctx, idx)
+	require.NoError(t, err)
+
+	rowIdx, colIdx, err := idx.Coordinates(shwap.EdsIDSize)
```

Comment:
This is supposed to be the size of the actual square, so `acc.Size`

Reply:
Fixed, thanks.